### PR TITLE
Fixed sound list padding and border style

### LINKED ISSUE #52: ui: redesign implementation: sound list
Update the sound list's styles so that it matches the UI design

Screen Shot 2020-09-19 at 1 25 32 PM

### PR DESCRIPTION
**Pull request by akshara-sun**
closes #52 
To-do: 

- [x] fix top and bottom border styles

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -35,15 +35,16 @@ h1, h1, h3, h4, h5, h6 {
   @apply bg-secondary text-primary;
 }
 
-.playlist {
+/* .playlist {
```
Comment (amilajack) [inline src/styles.css:38]: We need to remove all comments before merging

**Review by amilajack — CHANGES_REQUESTED** [review]

**Comment by amilajack** [comment]
The soundlist needs to have a border around each sound in the sound list

## Implementation
Screen Shot 2020-10-20 at 8 56 17 PM
## Design
Screen Shot 2020-10-20 at 8 55 51 PM

**Review by amilajack — CHANGES_REQUESTED** [review]
The soundlist needs to have a border around each sound in the sound list

## Implementation

Screen Shot 2020-10-20 at 8 57 30 PM

## Design

Screen Shot 2020-10-20 at 8 55 51 PM

**Comment by akshara-sun** [comment]
@amilajack 

**Thanks for catching that mishap with the border color in dark mode. I've got that part working (as you can see below) except now the top border is a problem. Working on it now, will keep you posted.**

image

**Comment by akshara-sun** [comment]
closes #52 

Current Sound List UI:
- Default (Light Mode):
image

- Dark Mode:
image

**Comment by akshara-sun** [comment]
I just realized the current styles might potentially pose problems to the design of our home page if and only if we decide to showcase a sound list of sorts on it. Maybe we can talk about this in our Saturday hack - just a thought.

@amilajack @hamirmahal

**Review by amilajack — APPROVED** [review]
Looks good to me! Nice job 🎉

**Comment by hamirmahal** [comment]
The sound list interface looks awesome, in both themes. Nice job!

**Comment by hamirmahal** [comment]
Also, I really appreciate the screenshots posted here for quick reference. It allows people to instantly see the results of changes from this pull request, without having to click on any additional links, which is really, really convenient. Thanks!

**Pull request merged**